### PR TITLE
[Copy] Update search page

### LINKED ISSUE #7718: 📝 Copy changes to the search page
## 📄 Copy Requirements
Talent seekers (manager users) need to get the correct information from the copy on the search page.

## Details
The text at the top of the search page needs to be update as follows:

### How to use this tool section new text

> **How to use this tool**
> Use the filters to specify your requirements. We will show you an estimated number of candidates who match your criteria as you enter your information. Select **Request candidates** when you are done. Doing so will bring you to a form where you can provide your contact information and submit your request.

## Screenshot
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBb09KIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--9ca233e401ffb1e5cdf14db653be92c75ba67750/image.png)

## ✅ Acceptance Criteria

Set of assumptions verifying that the issue was addressed when tested.

- [ ] H1 updated
- [ ] Centered div at top of page updated
- [ ] How to use this tool section updated
- [ ] Equivalent French updates are made
- [ ] Design review before merge (Inc French copy) @RM-1978  
- [ ] Change shared copy on request page too

### PR DESCRIPTION
**Pull request by vd1992**
🤖 Resolves #7718

## 👋 Introduction

Changes the copy on the search and a bit on the request pages. 

## 🕵️ Details

Needs to be sent for French translation
Needs design review after translations are received 

## 🧪 Testing

1. Navigate to search page and observe copy lines up with issue's request, no errors appear present

## 📸 Screenshot

image

### CODE REVIEW

**Review by petertgiles — APPROVED** [review]

**Comment by petertgiles** [comment]
Nice!  :+1: Don't forget your design review after the translations are in.

**Comment by vd1992** [comment]
@RM-1978 
The copy and its translation is in, how would you like to review it?

**Comment by RM-1978** [comment]
@vd1992, Thanks for taking the time to review with me.  As we said, 

1. The only change that I would make in English  would be to  place r "Request candidates" in quotations and mark up this phrase with the <strong> tag, to emphasize  that this text is the name of the links that managers should activate to move to the next step in the process.  
2. My comments on the French can be found in the FR JSon file :)

**Comment by vd1992** [comment]
> @vd1992, Thanks for taking the time to review with me. As we said,
> 
>     1. The only change that I would make in English  would be to  place r "Request candidates" in quotations and mark up this phrase with the ** tag, to emphasize  that this text is the name of the links that managers should activate to move to the next step in the process.**
> 
>     ** * My comments on the French can be found in the FR JSon file :) **

@RM-1978 
I can't see your comments on the file. Did you submit them?
If you picked "Start a review", you have to submit the review after you finish adding your comments.

image

Whereas 'Add single comment' lets you comment on something specific one-by-one

File: apps/web/src/lang/fr.json [hunk 1]
```diff
@@ -2575,6 +2567,10 @@
     "defaultMessage": "Mettre à jour l’utilisateur",
     "description": "Page title for the user edit page"
   },
+  "F+LDbs": {
+    "defaultMessage": "Si vous êtes à la recherche de talent, vous êtes au bon endroit. Notre base de données sur les talents est accessible à la plupart des ministères et organismes. Veuillez remplir la demande pour trouver des candidats qualifiés. Les candidats sont évalués selon leurs compétences, et regroupés en bassins pour répondre à vos besoins en matière de dotation.",
```
Comment (RM-1978) [inline apps/web/src/lang/fr.json:2571]: I would revise this text to read: "Si vous êtes à la recherche du talent, vous avez frappé à la bonne porte. Notre base de données de talents est ouverte à la plupart des ministères et des agences. Veuillez remplir une demande pour trouver des candidats qualifiés. Les candidats sont évalués selon leurs compétences et regroupés en bassins pour répondre à vos besoins en matière de dotation."

File: apps/web/src/lang/fr.json [hunk 2]
```diff
@@ -4620,6 +4616,10 @@
     "defaultMessage": "Désarchiver",
     "description": "Text on a button to un-archive the pool"
   },
+  "U0b3/+": {
+    "defaultMessage": "Utilisez les filtres pour préciser vos exigences. Nous vous montrerons un nombre estimatif de candidats qui répondent à vos critères, au fur et à mesure que vous entrez votre information. Veuillez sélectionner Obtenir des candidats une fois que vous aurez terminé. En ce faisant, vous verrez un formulaire à partir duquel vous pouvez fournir vos coordonnées et soumettre votre demande.",
```
Comment (RM-1978) [inline apps/web/src/lang/fr.json:4620]: I would revise this text to: "Utilisez les filtres pour préciser vos exigences. Nous vous montrerons un nombre estimatif de candidats qui répondent à vos critères, au fur et à mesure que vous entrez votre information. Veuillez sélectionner **Demande de candidats.** Vous verrez alors apparaître un formulaire à partir duquel vous pourriez fournir vos coordonnées et soumettre votre demande."

File: apps/web/src/lang/fr.json [hunk 3]
```diff
@@ -1719,6 +1719,10 @@
     "defaultMessage": "Ce formulaire vous permet de modifier une expérience en particulier, ses détails et les compétences qui s’y rattachent. N’oubliez pas que dans vos expériences de travail, vous devez décrire vos expériences de travail à temps partiel, de travail à temps plein, de travailleur autonome, de travail en tant que boursier, de travail au sein d’un organisme sans but lucratif ou de stagiaire.",
     "description": "Instructions on how to add an experience to your career timeline"
   },
+  "9Jkoms": {
+    "defaultMessage": "Trouver un talent numérique",
```
[the remark this answers is not in the document]
Comment (vd1992) [reply apps/web/src/lang/fr.json]: Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/7889/commits/909d5580ed79097136a94fd4f084e52076669f52

File: apps/web/src/lang/fr.json (hunk 1 above)
Comment (vd1992) [reply apps/web/src/lang/fr.json]: Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/7889/commits/909d5580ed79097136a94fd4f084e52076669f52

File: apps/web/src/lang/fr.json (hunk 2 above)
Comment (vd1992) [reply apps/web/src/lang/fr.json]: Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/7889/commits/97058b2a212bd83e40729fad788c06076e5664ca

Comment (RM-1978) [reply apps/web/src/lang/fr.json]: Review done. Changes have been reflected.

**Review by petertgiles — APPROVED** [review]
Noice

**Pull request merged**